tests/test_integration.py

```diff
@@ -22,7 +22,7 @@ def setUp(self):
             'merge': os.path.join(self.tmp_dir, 'merge'),
             'replace': os.path.join(self.tmp_dir, 'replace'),
         }
-        for work_dir_id, work_dir_path in self.work_dirs.items():
+        for _work_dir_id, work_dir_path in self.work_dirs.items():
```

Comment:
```suggestion
        for _, work_dir_path in self.work_dirs.items():
```

Reply:
I always leave the name on purpose because it makes it easier to see what the function returns if I ever need it again. But it's just my personal preference. I changed it according to your suggestion.

Reply:
This is already one by the way: c464ee24d0b598e26b348ad46bde33f62e21d3a8